script/upload-to-gcs.js

```diff
@@ -1,6 +1,6 @@
-const assert = require('assert')
```

Comment:
so... the standard for python is single quotes. do we have to use double quotes for javascript and single quotes for python? I would love if we could pick one or the other.

Reply:
Yeah sorry this was copy-pasted. You ok with double?

Reply:
love double

Reply:
#32